content/en/account_management/billing/custom_metrics.md

```diff
@@ -5,68 +5,199 @@ aliases:
   - /integrations/faq/what-standard-integrations-emit-custom-metrics/
 ---
 
-## Overview
+If a metric is not submitted from one of the [350+ Datadog integrations][1] it's considered a [custom metric][2]<sup>[(1)](#standard-integrations)</sup>.
 
-In general, [custom metrics][1] refers to any metrics that are not part of the regular integration suite, for example using custom checks or API-level metrics in your application stack.
+**A custom metric is uniquely identified by a combination of a metric name and tag values (including the host tag)**:
```

Comment:
The colon at the end of this sentence implies there's something to be something more? What was accidentally deleted here?

Reply:
nothing it's just moving content around that lead to this glitch :) just fixed it